### ISSUE TITLE
[autogsdoc] doc file generation date

### ISSUE THREAD

**Issue by rmottola** [gnustep/libs-base#328]
Currently, autogsdoc tries to be smart and puts as dates those found in the header (.h first, .m if not present). It scavenges for Date, Copyright and if present Revision too. However the latter one is CVS info, not really useful.
These dates are referred to the file content.

I think somewhere (maybe at the bottom or top of the file in a different style) a generation file of the doc itself should be present. This informs the user on when the generation was run, independently of when the class was last touched itself (and maybe somebody forgot to update its header copyrights).

What do you think? @rfm @bheron

**Comment by rfm** [comment]
Yes, this stuff seems obsolete (and perhaps never very useful to start with).
I agree that having a documentation generation timestamp (the date/time at which the headers/source were parsed) might be nice.

**Comment by rmottola** [comment]
Current progress is good and seems to work.
We need to clean up version files around, after being done with base. (Reminder).

Also, a clean solution for index.html would be interesting. E.g. min/max of dates + list of authors coming from all files.

**Comment by rfm** [comment]
It would be good to close this issue:

A min/max of dates is not readily do-able, since these are free text in any form desired by the author of the documentation.

A generation timestamp can be trivially provided with +[NSDate date] at the point when we generate the html.

A list of authors is trivial ... the -[AGSIndex authors] method provides that,  so you could get a comma separated list with a single line of code:

[[[[index authors] allKeys] sortedArrayUsingSelector: @selector(compare:)] componentsJoinedByString: @", "]

So we could put a list of authors and a timestamp at the bottom of each html document ... I think there's probably more effort involved in designing the html of such a footer than there is in gathering the data to populate it.

**Comment by rmottola** [comment]
Current situation:

- copyright dates are taken from the header file
- generation date is added
